### DISCUSSION TITLE
SMTP Provider list

### DISCUSSION THREAD

**Discussion by 0perationPrivacy** [0perationPrivacy/VoIP#59] [Ideas]
Here's a list of providers that provide SMTP (with at least a free tier). With each provider below, the following was used at account sign up:
- A **VPN**
- An **email alias** service _(Anonaddy/SimpleLogin/33mail)_
- An **alias name**

If for any reason the provider rejects the account creation you will see the criteria for getting banned in the Notes column.

Please comment if you find a better service.
_(this is for sms-to-email notifications. You will configure the SMTP credentials of your choice. This list will provide guidance)_

Service | Notes
-- | --
❌ SendGrid | Banned for using a VPN, heavy questionnaire for creating a free account. No longer supports smtp as of 2022, only API keys
✅ Gmail | Can create an account over VPN using a burner sim or Telnyx non-US number
❌ SendPulse | Blocked for providing anonymous info, using VPNs, etc
❌ MailGun | works but info.mailgun.com is categorized as a Phishing link. Malware defenses block it or send it to spam
✅ [smtp2go](https://www.smtp2go.com/) | VoIP numbers not allowed warning, but accepts a UK Telnyx number. SimpleLogin alias email accepted with an alias name. Free account allows 1,000 emails per month! Works great so far.
✅ PostMark ✅ | (testing) SimpleLogin Premium worked (100/m) Looks more private - instant access - Clean VPN IP
✅ SendinBlue | (testing) logo in footer in free version (300/day)
🙄 MooSend | need an active website to verify
❌ Elastic Email | Can’t use burners. Leaves a long unsubscribe footer.
💔 Amazon SES | free only if you send from amazon, not smtp. Otherwise $0.10/1,000 emails
❌ TurboSMTP (serverSMTP.com) | Rejected. too much info required, they verify email and address.
❌ MailJet |  Need ID to verify. Logo in email
❌ Easy-SMTP | Account creation unavailable
💔 MailGet SMTP (pabbly) | only built-in mailer, no smtp
🙄 SparkPost | Banned. 500 emails, requires website
❌ SocketLabs | SimpleLogin Premium BANNED (must own domain. 2,000/m) - [30-45 min wait time: setup server] Not enough information provided - Clean VPN IP
✅ Yandex | Limits 50 emails/day to prevent spam

**Comment by 0perationPrivacy** [comment]
**Note:** I have self hosted mail-in-a-box on a VPS with SMTP credentials without issue for years. But it takes time to configure, update, maintain, troubleshoot. Not for everyone.

**Comment by fightforprivacy** [comment]
I like the table structure, but can we add more rows?
Such as
VPN
Used email alias to sign up
Required verification via email
Required verification via phone number
Required verification via ID
Free tier - max email sends per day
Free tier - calendar use
Free tier - storage
Free tier - location restricted

**Reply by 0perationPrivacy** [reply]
I really wanna keep this **simple** (K.I.S.S), too many columns and now it gets too complicated to maintain 🤪. If account creation is allowed, then no exceptions are mentioned. Like with gmail. If there are issues, only those are called out in the notes.
But a default given is mentioned in the beginning: **VPN, Alias Email, Alias name**. That's the criteria, no need to repeat it for each row IMO. I'll update it with users comments and experiences and will add columns as it makes more sense.

**Reply by 0perationPrivacy** [reply]
VPN is a given, all accounts are made with a VPN, an alias email, etc, that's min requirements, if it gets rejected, no point in going any further, then the rest of the features don't even matter.

**Reply by fightforprivacy** [reply]
All really valid feedback. 

Could we then update the description


```
Here's a list of providers that provide SMTP (with at least a free tier). With each provider below, the following was used at account sign up;
- A VPN
- An email alias service (Anonaddy / SimpleLogin)
- An alias name

If for any reason the provider rejects the account creation you will see the criteria for getting banned in the Notes column.
```

**Reply by fightforprivacy** [reply]
Updated the grammar a bit

**Reply by 0perationPrivacy** [reply]
updated, **thanks**. Give it time, eventually we'll test all. This list will be updated as we test and report back results.

**Reply by fightforprivacy** [reply]
Can I get an email for you two? 

I want to discuss something privately.

**Reply by fightforprivacy** [reply]
Great, thanks!

**Reply by monkeysac** [reply]
@fightforprivacy 
> Domain Creation Date: 2021-10-10  👀 👍 

Call me intrigued...what might there be in store?

**Reply by fightforprivacy** [reply]
> 
> 
> @fightforprivacy
> 
> > Domain Creation Date: 2021-10-10  👀 👍
> 
> Call me intrigued...what might there be in store?

I'm intrigued, @monkeysac, where did you get that?

**Reply by fightforprivacy** [reply]
Ahh, my github bio, thats where you got the domain from, then you just got the  date from one of many places online. Neat.

**Reply by monkeysac** [reply]
> Ahh, my github bio, thats where you got the domain from, then you just got the date from one of many places online. Neat.

Lol, yes...no digging involved.  Figured it's a project you're working on and public knowledge considering it's on your bio.  Since we're all here to do essentially as the domain implies I figured it may be something the community would benefit from.  Let me know if you'd rather me delete my reply so you can wipe your profile and I'm sure @privacyosint would do same.

**Reply by fightforprivacy** [reply]
I appreciate it. It is public knowledge, or atleast will be soon.

**Reply by monkeysac** [reply]
> I appreciate it. It is public knowledge, or atleast will be soon.

Awesome, looking forward to it.  I'd love to learn whether or not it's an effort I can contribute to. 202.dt8bk@8shield.net

**Comment by monkeysac** [comment]
i understand anon smtp may be tough to come by but might i suggest more research (or a limited selection of providers) in favor of including gmail?  it's impossible to me to imagine zero knowledge with *anything* traversing google ip space...

**Reply by 0perationPrivacy** [reply]
Totally understood. NONE of the options above are totally private. The only private solution is self hosting a mail server on a VPS that allows port 25. I have MIAB (Mail-in-a-box) set up but requires hours and a huge learning curve. (you also need to consider private domain hosting, private name server and DNS provider, etc)

But like all the rest of the options, if a Gmail account can be created with all anonymous details (I have plenty of those), then it falls in the same category. We're only using SMTP. Most, if not **all** of these services which are free, will have trackers just like google (probably google analytics) and will harvest as much info from you as possible (browser fingerprinting, ads, trackers in the email footer, IPs, user agents, etc).

That's why this thread was created, for continuous research by anyone viewing this. The platform will allow any SMTP, we just have to pick and chose the right provider. The research here is to help users make the right choice and not waste their time with providers already researched.

My suggestion for the most anonymous option is `Mail-in-a-box`. But it's not for the common person and hosting on a VPS is not free.

**Reply by monkeysac** [reply]
>The only private solution is self hosting a mail server on a VPS that allows port 25
> My suggestion for the most anonymous option is `Mail-in-a-box`. But it's not for the common person and hosting on a VPS is not free.

Yes for sure. I self-host but not using MIAB. I look forward to the day there's a self-hosted Proton/Tuta-esque email solution for encryption at rest + decrypt in user DOM.  It's been a few months since I've researched it but I assume nothing exists yet (?)

To others giving consideration to standing up self-hosted email, as @0perationPrivacy its not for the faint of heart but it's a fun project.  I recommend using the [guide](https://github.com/anonaddy/anonaddy/blob/master/SELF-HOSTING.md) put together by @willbrowningme for [anonaddy](https://github.com/anonaddy/anonaddy) if you're interested.

**Reply by 0perationPrivacy** [reply]
Thanks, you should def look into MIAB, installation is a few commands. Not nearly as long as Anonaddy.
I prefer self hosting SimpleLogin as it has better options. Also takes in a PGP key from protonmail, but that's for email forwarding and not SMTP. There's a ticket open for that but it doesn't seem like they'll be getting in the business of a mail server.

**Reply by monkeysac** [reply]
I went the manual route with anonaddy for the learning experience...certainly there are more automated solutions :)  Simplelogin is good too...i switched from that to anonaddy for $reasons_i_dont_remember. I was an early adopter so it may be worthy of a revisit.

**Comment by ghost** [comment]
Hello, I apologize if this is not the place to ask for help, but I am trying to setup notifications using PostMark and it's not working (I must be doing something wrong).   I signed up for a PostMark account and followed their [guides for SMTP](https://postmarkapp.com/developer/user-guide/send-email-with-smtp).  I created an SMTP token because there is no place in the VoIP Suite Email Settings to put in the header, which is what is required when using a Sever Token.  I created an SMTP token which provides an Access key which is the username and a Secret key which acts as the password.  The details that I put in are below.  I click Save and it says everything was saved successfully.  I then logout of VoIP Suite and send some test texts.  I can send and receive text messages but I don't receive any notifications via email when logged out.  Unsure of where to start troubleshooting.  I am using a simple login email address as the to address - not sure if that matters.  Any help would much appreciated.  This is the last step for me in the process.  I have setup everything else and it works great!  This is so cool!

Email Settings:
- Username:  SMTP token Access Key
- Password:  SMTP token Secret Key
- From Address:  email address setup in PostMark
- To:  same as from address
- Server: smtp.postmarkapp.com
- Port: 587
- Secure: unchecked

**Reply by ghost** [reply]
Okay, I figured out what I missed.  I needed to select the profile [checkbox] which is below the email settings.  Once I did that, everything worked great!  I knew it had to be something easy.
